### DISCUSSION TITLE
RTC

### DISCUSSION THREAD

**Discussion by Bonusmp** [edterbak/HeishaMoNR#138] [Q&A]
hello,

I have a problem with the RTC, the switch-on temperature is always higher than the switch-off temperature. And higher to room set point

![obraz](https://github.com/edterbak/NodeRed_Heishamon_control/assets/75584351/79818365-e054-4f7c-830c-5737aab58451)
![obraz](https://github.com/edterbak/NodeRed_Heishamon_control/assets/75584351/57fca173-e359-42f2-9204-53325ddc5a3a)
![obraz](https://github.com/edterbak/NodeRed_Heishamon_control/assets/75584351/8b7c2fdc-2bf1-4ae0-9ffe-0992a3c37d7e)

**Comment by edterbak** [comment]
Where have you installed node red? In ha or Linux.
Have you correctly set the persistent storage?
Might be related to that.

Otherwise, what happens if you change toggles and values on RTC tab, and on home dashboard for war fuction / water temperature

**Reply by Bonusmp** [reply]
I have an installation on HA, as a direct system

<img width="447" alt="image" src="https://github.com/edterbak/NodeRed_Heishamon_control/assets/75584351/4a72c246-b5e2-43a6-9da2-cec7ec691ced">

<img width="757" alt="image" src="https://github.com/edterbak/NodeRed_Heishamon_control/assets/75584351/0d4c9a94-053c-429c-a1dd-866614c8f5ce">

**Reply by edterbak** [reply]
Okay, it seems not to be the storage. :) goooood..

You have the pump set in Direct modus, and you are using a thermostat?

If so, can you try to set the pump zone control to "WaterTemperature". 23.09 stable is not playing nice with thermostats.  
In the latest beta (flow 23.10 beta9) I have improved the functionality of thermostat internal/external and roomsensor a lot. I hope this version will come out of beta soon so this kind of error are prevented if this is the case. It is not ready for big audience just yet. I hope to have it ready in a week or two.

Back to the problem.... Did you try to set the trigger / revert temperature to 0,2 and 0,1? does that work?

**Reply by Bonusmp** [reply]
I have the pump controlled by Panasonic WAR, and I added a room temperature sensor (sensor from the recuperator connected to HA). I wanted the pump not to turn on when the temperature inside the living room is OK.

Additionally, it will limit the number of starts (my pump is too large compared to the building's needs). I was also thinking about a function that would block the pump from turning on, e.g. for 2 hours, if three consecutive turns on were shorter than 30 minutes each.

I will check the given settings, one question about the room set point - is this not the temperature expected in the room?

**Reply by edterbak** [reply]
> one question about the room set point - is this not the temperature expected in the room?

Yes it is. It is the desired room temperature.  
You are using the Panasonic with 'compensation curve' mode, correct?

**Reply by Bonusmp** [reply]
Yes

**Comment by edterbak** [comment]
I changed the logic for the two temperatures.
They can now be egual to eachoter. 

will be in next beta from now.

**Reply by Bonusmp** [reply]
now its work correct ;)

**Reply by edterbak** [reply]
Thats fast 😄
